prettyprinter/prettyprinter.cabal

```diff
@@ -36,6 +36,10 @@ library
         , Prettyprinter.Render.Util.SimpleDocTree
         , Prettyprinter.Render.Util.StackMachine
         , Prettyprinter.Util
+        , Prettyprinter.Util.Compat.Text
+        , Prettyprinter.Util.Compat.Text.IO
+        , Prettyprinter.Util.Compat.Text.Lazy
+        , Prettyprinter.Util.Compat.Text.Lazy.Builder
```

Comment:
These will show up on Hackage, cluttering the package overview quite a bit – half of the whole package is already compatibility stuff. I don’t think there’s a way around this, or is there?

Reply:
Perhaps one option is to move these to a separate package altogether?